### PR TITLE
Add a runtime module for Kotlin

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Review by gsmet — CHANGES_REQUESTED** [review]
Shouldn't the `deployment` module have a dependency to the `runtime` module?

Even if empty for now, that might not be always the case in the future.

**Comment by geoand** [comment]
Aren't the deployment modules brought in automatically? If not, I will of course add it :)

**Comment by gsmet** [comment]
I was talking about having a dependency from the `deployment` module to the `runtime` one. I think that's still what we're supposed to do nowadays:
https://github.com/quarkusio/quarkus/blob/master/extensions/agroal/deployment/pom.xml#L41

I agree it's not useful right now for this specific module but let's be consistent.

**Comment by geoand** [comment]
@gsmet :+1, fixed :)

File: extensions/kotlin/runtime/pom.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-kotlin-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-kotlin</artifactId>
+    <name>Quarkus - Kotlin</name>
```
Comment (gsmet) [inline extensions/kotlin/runtime/pom.xml]: Minor nitpick, sorry for not spotting this earlier, I think all the others are suffixed ` - Runtime`. Not sure if we will keep it but better be consistent for now.

Comment (geoand) [reply extensions/kotlin/runtime/pom.xml]: Good catch! Fixed

**Comment by geoand** [comment]
The test failures seems unrelated... something with Camel?

**Comment by geoand** [comment]
Tests are now passing after I manually restarted them.

**Review by gsmet — APPROVED** [review]

**Pull request merged**